### ISSUE TITLE
Test: Name of server definition file

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbem#1608]
The server definition file introduced with the end2end tests may need a rename. See the respective comments on PR #1606.

The current names are:
```
"server definition file"               # Term for the file
tests/servers/server_file.yml          # Server definition file (not in repo)
tests/servers/server_file_example.yml  # Example for server definition file
tests/servers/server_file.py           # Module encapsulating access to the file
```

**Discussion point:** Proposals for better term and file names.

**Comment by andy-maier** [comment]
We agreed to use `server_definition_file` in the file names, and keep the term "server definition file".

**Comment by andy-maier** [comment]
Fixed in PR #1621.
